com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zcl/ZclCluster.java

```diff
@@ -1352,8 +1329,7 @@ private void handleDiscoverAttributes(DiscoverAttributesCommand command) {
             }
         }
 
-        DiscoverAttributesResponse response = new DiscoverAttributesResponse();
-        response.setAttributeInformation(attributeInformation);
+        DiscoverAttributesResponse response = new DiscoverAttributesResponse(true, attributeInformation);
```

Comment:
This should not be ```true``` - I'm creating a PR to fix a bug here which we should fix in this PR as well.